src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import '../../App.scss';
+
+export const Person = ({ name, age, sex, isMarried, PartnerName }) => {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age !== undefined
+        && <p className="Person__age">I am {age}</p>
+      }
```

Comment:
```suggestion
      {!!age
        && <p className="Person__age">I am {age}</p>
      }
```
Type check is not a better way for conditions.

Reply:
What about age = 0 ? Then we wouldn't have the right answer.